<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorWidget</class>
 <widget class="QWidget" name="ScriptEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="m_pStartButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="m_pStopButton">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="m_pOpenButton">
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="m_pSaveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="m_pSaveAsButton">
     <property name="text">
      <string>Save As...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="m_pTextEdit">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
